tests/testthat/test-run_trisk.R

```diff
@@ -53,20 +49,18 @@ test_that("with loans, without iteration, using minimum values of input argument
   expect_snapshot(lapply(results, as.data.frame))
 })
 
-test_that("with equity, without iteration, using maximum values of input arguments output is unchanged", {
+test_that("without iteration, using maximum values of input arguments output is unchanged", {
```

Comment:
I would suggest remoing this test and instead make the snapshot test for run_lrisk use the maxium values. Simply yo save time when running the tests.